package.json

```diff
@@ -119,7 +119,7 @@
       "git add"
     ],
     "*.go": [
-      "gofmt -w -s",
+      "gofmt -w -s pkg",
```

Comment:
I dont think this works as expected. husky adds the filepath to the end of this cmd. So this becomes 
`gofmt -w -s pkg /what/ever/files/you/changes.go`
I think a better fix would be to only check go files in `pkg/**.go`

Reply:
Good catch. Will fix in master straight away!

Reply:
Good catch. Will fix in master straight away!

Reply:
No rush :)